packages/@aws-cdk/core/lib/stack-synthesizers/default-synthesizer.ts

```diff
@@ -453,6 +459,11 @@ export class DefaultStackSynthesizer extends StackSynthesizer {
       requiresBootstrapStackVersion: MIN_BOOTSTRAP_STACK_VERSION,
       bootstrapStackVersionSsmParameter: this.bootstrapStackVersionSsmParameter,
       additionalDependencies: [artifactId],
+      lookupRole: this.lookupRoleArn ? {
```

Comment:
It's probably useful if people can disable this. I can imagine environments in which the lookup role has been neutered, or people prefer using current credentials to access their stacks.

Add a property to switch this off?

Reply:
Add a `disableLookupRole` property.